blog/2022-01-24-new-name-new-website/index.md

```diff
@@ -0,0 +1,32 @@
+---
+authors: [doris]
+tags: [website, documentation, branding, docusaurus]
+---
+
+# New Name, New Website!
+
+**This past month has been quite the roller coaster for us at [Some Engineering](https://some.engineering).** 🎢
+
+Right before Christmas, it was decided that we needed to part ways with the name "Cloudkeeper." There were many reasons behind this decision, one of which was that we did not feel the product we were developing should be limited to cloud infrastructure. We were building a product that did much more than just "housekeeping for your clouds."
+
+We partnered with brand namer [Steve Cecil](http://wherewords.com) to devise a new name. I can't speak for the rest of the naming committee, but I found the process to be really special and quite fun (though [Steve](http://wherewords.com) did require us to invest quite a bit of our time).
```

Comment:
"invest quite a bit of time" --> I feel we should always be specific. "a lot", "a little" - how much time exactly is that?

in this case, maybe:

[...] to invest quite a bit of our time. We had two separate 2-hour working sessions. For the second session, the homework for each one on our "naming committee" was to review 230+ potential names that Steve had created for us, and to create a shortlist of our 20 favorite names.